oaknational/Oak-Web-Application
# fix(seo): add no-index to seo

## Description

- List of changes
Change <meta name="robots" content="index,follow"/> to <meta name="robots" content="noindex" /> across all /beta pages

## Issue(s)

Fixes #1451 

## How to test

1. Go to beta pages, open dev tools, elements
2. Click on dev tools, elements
3. You should see the lines  <meta name="robots" content="noindex" /> 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
